### PR TITLE
Update ReadMeOV.rst

### PR DESCRIPTION
**Pull request by vthaniel**
### Description
Updated pypi Wheels ReadMe to add NPU device. Updated OV version

### Motivation and Context
New OVEP 1.18 python wheels officially supports NPU (Windows only) and OV2024.1

### CODE REVIEW

File: docs/python/ReadMeOV.rst [hunk 1]
```diff
@@ -15,26 +16,27 @@ Requirements
...
  - Intel® discrete GPUs
+ - Intel® integrated NPUs (Windows only)
 
 ``pip3 install onnxruntime-openvino``
 
 Please install OpenVINO™ PyPi Package separately for Windows.
 For installation instructions on Windows please refer to  `OpenVINO™ Execution Provider for ONNX Runtime for Windows <https://github.com/intel/onnxruntime/releases/>`_.
 
-**OpenVINO™ Execution Provider for ONNX Runtime** Linux Wheels comes with pre-built libraries of OpenVINO™ version 2023.0.0 eliminating the need to install OpenVINO™ separately. The OpenVINO™ libraries are prebuilt with CXX11_ABI flag set to 0.
+**OpenVINO™ Execution Provider for ONNX Runtime** Linux Wheels comes with pre-built libraries of OpenVINO™ version 2024.1.0 eliminating the need to install OpenVINO™ separately. The OpenVINO™ libraries are prebuilt with CXX11_ABI flag set to 0.
```
Comment (sfatimar) [inline docs/python/ReadMeOV.rst:32]: Please remove the ABI Flag line

**Comment by sfatimar** [comment]
Please merge this PR in ovep-develop-lnl-1.1

**Comment by vthaniel** [comment]
Closing this PR. Since the base has been changed, there are additional unintended commits in this PR

**Comment by vthaniel** [comment]
Closing this PR. Since the base has been changed, there are additional unintended commits in this PR

**Pull request closed without merging**